spec/controllers/categories_controller_spec.rb

```diff
@@ -40,15 +40,28 @@
 
   describe 'Post #create' do
     context 'when valid' do
+      subject(:create_category) { post :create, params: params }
+
+      let(:title) { Faker::Internet.user_name(5..15) }
+      let(:params) do
+        {
+          category: {
```

Comment:
Maybe it's better to move `{` to a new line 🤔

Reply:
@sinpai I think there is no difference